proxy/envoy/testdata/envoy-mixer-auth.json.golden

```diff
@@ -0,0 +1,51 @@
+{
+  "listeners": [
+    {
+      "address": "tcp://0.0.0.0:15004",
+      "name": "tcp_0.0.0.0_15004",
+      "filters": [
+        {
+          "type": "read",
+          "name": "tcp_proxy",
+          "config": {
+            "stat_prefix": "tcp",
+            "route_config": {
+              "routes": [
+                {
+                  "cluster": "in.9091"
+                }
+              ]
+            }
+          }
+        }
+      ],
+      "ssl_context": {
+        "cert_chain_file": "/etc/certs/cert-chain.pem",
```

Comment:
This is what i am talking about in previous comment. It is the listener where we put the ssl_context, not cluster.